config/initializers/github_omniauth.rb

```diff
@@ -1,3 +1,3 @@
 Rails.application.config.middleware.use OmniAuth::Builder do
-  provider :github, ENV["github_client_id"], ENV["github_client_secret"]
+  provider :github, ENV["github_client_id"], ENV["github_client_secret"], {:provider_ignores_state => true}
```

Comment:
Line is too long. [107/80]<br>Redundant curly braces around a hash parameter.<br>Use the new Ruby 1.9 hash syntax.<br>Space inside { missing.<br>Space inside } missing.